src/lib/common/JsonHelper.h

```diff
@@ -0,0 +1,68 @@
+#ifndef SRC_LIB_COMMON_JSONHELPER_H
+#define SRC_LIB_COMMON_JSONHELPER_H
+
+/*
+*
+* Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+
+#include <sstream>
+#include <string>
+#include <vector>
+
+class JsonHelper
+{
+public:
+  JsonHelper();
```

Comment:
Json is too broad in my opinion. JsonHelper and JsonRender are also good names. Maybe we can keep the current name and adjust in the future if we find the class adquires new semantics in the future and the name is no longer correct (renaming class is pretty easy, see https://github.com/telefonicaid/fiware-orion/issues/1173).